schema.sql

```diff
@@ -0,0 +1,179 @@
+DROP DATABASE IF EXISTS lairdnd;
```

Comment:
Please fix this test here.

Reply:
As the comment above was a test comment, resolving now.